### ISSUE TITLE
[Accessibility]DateTimePicker control should support Tab stop for its drop-down button

### ISSUE THREAD

**Issue by Jenna-Zheng** [dotnet/winforms#3613]
* .NET Core Version: 
.NET SDK 5.0.100-rc.1.20367.2

* Have you experienced this same bug with .NET Framework?: 
Yes

**Problem description:**
There is no tab stop for DateTimePicker drop-down button, so user don't know how to expand this control's calendar pane, but actually we can use Alt+Down arrow or press F4 keyboard to expand.
![image](https://user-images.githubusercontent.com/64949616/88007264-2e19a700-cb40-11ea-84d5-8645bb6fd052.png)
![TestResult_keyboard](https://user-images.githubusercontent.com/64949616/88007986-c3696b00-cb41-11ea-9086-afd239de17ba.gif)

**Expected behavior:**
DateTimePicker control should have a tab stop for its drop-down button while navigating through Left\Right arrow
![image](https://user-images.githubusercontent.com/64949616/88006855-3ae9cb00-cb3f-11ea-9b45-ac7c1ee00d85.png)

**Minimal repro:**
1. Create a .Net Core WinForms App.
2. Add a DateTimePicker control to the Form.
3. Build and run the app.
4. Navigate the DateTimePicker control by using Left\Right arrow.
5. Observe the keyboard focus.

**Comment by merriemcgaw** [comment]
This is nice-to-have, but since Alt+down arrow is the most common way to expand drop downs, and it works for this control (which reports to AT that it's a drop-down) the general Narrator user won't be too confused, hence not a MAS violation.

**Comment by weltkante** [comment]
Not sure if I like this suggestion, it would require non-accessibility users double the number of tab presses to go through some of our forms which have a lot of date fields. If this is implemented it would be nice to be able to opt-out or tie it this feature to accessibility.

**Comment by merriemcgaw** [comment]
That's a great point @weltkante. The other issue we may run into is that we may not have direct access to this button making it a Common Controls issue. @guybark, what are your thoughts on @weltkante's concern about the extra number of tabs?

**Comment by vladimir-krestov** [comment]
I agree with @weltkante, we shouldn't change users "Tab" expirience.
Also, Alt+down/F4 buttons work for the control. Moreover, DateTimePicker supports Expand/Collapse pattern, so Narator announces if the control is expanded/collapsed. So users know how to work with it. 
![image](https://user-images.githubusercontent.com/49272759/147466774-9991d6d4-7d6f-4f6d-b3dd-dc7417e40e1c.png)
In my opinion, we need to close this issue. @merriemcgaw, your thoughts?

**Comment by merriemcgaw** [comment]
Agreed!
